.hound.yml

```diff
@@ -1,4 +1,3 @@
-# Prefer single quotes
-StringLiterals:
-  EnforcedStyle: single_quotes
-  Enabled: true
```

Comment:
As far as I understand `hound-tools` documentation it does not touch ``hound.yml`, so these changes should be there. What is more information about excluding `schema.rb` should also be added here. 

I don't understand why `hound-tools` cannot take overrides from `.hound.yml` but take them from `.hound/overrides.yml`.